ocs_ci/deployment/deployment.py

```diff
@@ -971,6 +971,23 @@ def deploy_ocs(self):
             # Change registry backend to OCS CEPHFS RWX PVC
             registry.change_registry_backend_to_ocs()
 
+        # Enable console plugin
+        ocs_version = version.get_semantic_ocs_version_from_config()
+        if (
+            ocs_version >= version.VERSION_4_9
+            and config.ENV_DATA["enable_console_plugin"]
+        ):
+            logger.info("Enabling console plugin")
+            ocp_obj = ocp.OCP()
+            patch = (
+                '\'[{"op": "add", "path": "/spec/plugins", "value": ["odf-console"]}]\''
+            )
+            patch_cmd = (
+                f"patch console.operator cluster -n {constants.OPENSHIFT_STORAGE_NAMESPACE}"
+                f" --type json -p {patch}"
+            )
+            ocp_obj.exec_oc_cmd(command=patch_cmd)
```

Comment:
Logically looks good to me but I'd like to suggest we move this to a function we can call here instead. `deploy_ocs` is a pretty big method already and I'd like to limit the amount of code we are adding to it directly. If you agree this could definitely be taken up as a follow-up, no need to hold this PR up.

Reply:
ack. I will take it up as follow-up